modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/components/RuleBuilder/RulesSupport.es.js

```diff
@@ -14,6 +14,9 @@
 
 import {PagesVisitor} from 'dynamic-data-mapping-form-renderer';
 
+import {DEFAULT_FIELD_NAME_REGEX, EMPTY_FIELD_REGEX} from '../../util/regex.es';
```

Comment:
@alinedoleron, why aren't we reusing it [here](https://github.com/liferay/liferay-portal/blob/7600a72838817db1adc56ba435ee7af1222873f7/modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/test/js/components/LayoutProvider/LayoutProvider.es.js#L25) and [here](https://github.com/liferay/liferay-portal/blob/7600a72838817db1adc56ba435ee7af1222873f7/modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/test/js/components/LayoutProvider/handlers/fieldEditedHandler.es.js#L22)?